scripts/jenkins/ardana/ansible/roles/send_to_os_health/vars/tempest.yml

```diff
@@ -23,3 +23,4 @@ os_health_metadata:
   - "qe_env:{{ qe_env }}"
   - "tempest_filter:{{ tempest_run_filter }}"
   - "build_name:{{ os_health_build_name }}"
+  - "openstack_service:{{ tempest_run_filter }}"
```

Comment:
I would have expected something like tempest. Are you sure about the value?

Reply:
@JanZerebecki This PR is intended only for testing a requirement for Dashboard results. We would like to see results group by based on a openstack service.